deals/rpc/rpc.proto

```diff
@@ -53,47 +53,93 @@ message RetrievalDealRecord {
     RetrievalDealInfo deal_info = 3;
 }
 
-message DealConfig {
+message StorageDealConfig {
```

Comment:
I think we should notify the team building the python client about proto changes, since they're relying on them in their work.

Reply:
Yes, I'll let them know. They'll be able to pull in the generated python stubs from pypi at least, and I can give them a more detailed explanation as well.